src/quicreach.cpp

```diff
@@ -401,13 +397,13 @@ int QUIC_CALL main(int argc, char **argv) {
 
     MsQuic = new (std::nothrow) MsQuicApi();
     if (QUIC_FAILED(MsQuic->GetInitStatus())) {
-        printf("MsQuicApi failed, 0x%x\n", MsQuic->GetInitStatus());
+        std::cout << "MsQuicApi failed, 0x" << MsQuic->GetInitStatus() << std::endl;
```

Comment:
I don't see anything making this print the value in hex.

Reply:
I will fix this. Thank You ✌